runtime/src/in_mem_accounts_index.rs

```diff
@@ -468,6 +468,10 @@ impl<T: IndexValue> InMemAccountsIndex<T> {
     /// any entry at 'slot' is replaced with 'account_info'.
     /// or, 'account_info' is appended to the slot list if the slot did not exist previously.
     /// returns true if caller should addref
+    /// conditions when caller should addref:
+    ///   'account_info' does NOT represent a cached storage
```

Comment:
nit: Maybe some quick context of what this means

'account_info' does NOT represent a cached storage (the slot is being flushed from the cache)